canvas/models/models.py

```diff
@@ -132,6 +132,14 @@ def total_tokens_received(self):
                 tokens += 0
             else:
                 tokens += event.tokens_received(team)
+
+        from course.models.models import UserQuestionJunction
+
+        practiced_uqjs = UserQuestionJunction.objects.filter(user=self.user, is_solved=True, question__event=None)
+        for uqj in practiced_uqjs:
+            tokens += uqj.tokens_received
+        # print('token: ', tokens) # token: xx is printed multiple times, and eventually ends at 0. why?
```

Comment:
Check why this method is being called multiple times. Are they all for the same user? What are the uqjs when the token is 0?